test/cli/test_cli_functions.sh

```diff
@@ -196,6 +217,15 @@ function noobaa_install {
     test_admission_deployment
 }
 
+function noobaa_install_external {
+    #noobaa timeout install # Maybe when creating server we can use local PV
```

Comment:
We might add some printing here to make it clear that we are in external postgres mode.

Reply:
added print in install_external